site/installation/README.md

```diff
@@ -8,8 +8,108 @@ menu:
     weight: 1
 ---
 
-Users can install kpt as a [gcloud component](/installation/gcloud/), a 
-[brew tap](/installation/homebrew/), or from [source](/installation/source/).
+Users can get kpt in a variety of ways:
+1. Install with [GCloud](#GCloud)
+1. Install with [Homebrew](#Homebrew) on MacOS
+1. Use [Docker](#Docker)
+1. Download [Binaries](#Binaries) 
+1. Build from [Source](#Source)
 
-Alternatively, kpt can be [run in a docker container](/installation/docker/) or
-from [statically compiled go binaries](/installation/binaries/).
\ No newline at end of file
+## GCloud
+
+Install with gcloud.
+
+```shell
+gcloud components install kpt
+```
+
+```shell
+kpt version
+```
+
+The version of kpt installed using `gcloud` may not be the latest released version.
+
+## Homebrew
+
+Install the latest release with Homebrew on MacOS
+
+```shell
+brew tap GoogleContainerTools/kpt https://github.com/GoogleContainerTools/kpt.git
+brew install kpt
+```
+
+```shell
+kpt version
+```
+
+## Docker
+
+Use one of the kpt docker images.
+
+| Feature   | `kpt` | `kpt-gcloud` |
+| --------- |:-----:|:------------:|
+| kpt       | X     | X            |
+| git       | X     | X            |
+| diffutils | X     | X            |
+| gcloud    |       | X            |
+
+### [gcr.io/kpt-dev/kpt]
+
+```shell
+docker run gcr.io/kpt-dev/kpt version
+```
+
+### [gcr.io/kpt-dev/kpt-gcloud]
+
+An image which includes kpt based upon the Google [cloud-sdk] alpine image.
+
+```shell
+docker run gcr.io/kpt-dev/kpt-gcloud version
+```
+
+[gcr.io/kpt-dev/kpt]: https://console.cloud.google.com/gcr/images/kpt-dev/GLOBAL/kpt?gcrImageListsize=30
```

Comment:
Should these be at the bottom of the file?

Reply:
updating.  Not sure if it matters for MD but why not be consistent.